homeassistant/components/unifi/config_flow.py

```diff
@@ -179,7 +182,9 @@ def __init__(self, config_entry):
     async def async_step_device_tracker(self, user_input=None):
         """Manage the device tracker options."""
         if user_input is not None:
-            return self.async_create_entry(title="", data=user_input)
+            self.options.update(user_input)
+            return await self.async_step_statistics_sensors()
```

Comment:
two returns?

Reply:
Draft PR :)

Reply:
So what is happening here? Are we going to first show 1 screen, then another screen ?

Reply:
Since it's just one option, can't we put them in 1 screen ?

Reply:
Im planning to add other sensors later on, so I'd rather group them together. This is also why I think enabling descriptions would be a good improvement, describing the theme of the options.